src/snowflake/connector/cursor.py

```diff
@@ -988,7 +987,7 @@ def execute(
             if "data" in ret and "sqlState" in ret["data"]
             else None
         )
-        logger.info("query execution done")
+        logger.debug("query execution done")
```

Comment:
We updated it to debug because the INFO logs may have a lot of "query execution done" while there was no other information was printed.
In general I think it's too verbose to print INFO logs for each query because it would be too verbose. What do you think? But I'm wondering if some customers had requested it so we added it.

Reply:
I think something to indicate what's going on could be useful, but it's up to you!